out/models.go

```diff
@@ -10,10 +10,11 @@ type OutRequest struct {
 }
 
 type Params struct {
-	From string `json:"from"`
-	File string `json:"file"`
-	To   string `json:"to"`
-	Acl  string `json:"acl"`
+	From        string `json:"from"`
+	File        string `json:"file"`
+	To          string `json:"to"`
+	Acl         string `json:"acl"`
+	ContentType string `json:"content-type"`
```

Comment:
This should be `content_type` (underscore rather than hyphen) as that's how all other multi-word keys look in the pipeline config.